src/services/helpers/types.ts

```diff
@@ -62,4 +62,6 @@ export enum WorkspaceAction {
   DELETE_WORKSPACE = 'Delete Workspace',
   ADD_PROJECT = 'Add Project',
   ADD_CUSTOM_WORKSPACE = 'Add Workspace',
+  RESTART_WORKSPACE = 'Restart Workspace',
```

Comment:
After I tried to restart STARTING workpsace, I found two issues:

1. Not really related to your PR, Workspace loading page says Starting workspace, when it's stopping:
![Screenshot_20210402_093643](https://user-images.githubusercontent.com/5887312/113388642-46acb300-9397-11eb-8128-07e69b0f96e8.png)
2. After workspace start is interrupted, error is kept and we don't follow startup process:
![Screenshot_20210402_093652](https://user-images.githubusercontent.com/5887312/113388672-56c49280-9397-11eb-80d5-19692e0f8939.png)